react/AvailabilityNotifier.tsx

```diff
@@ -60,18 +60,20 @@ function AvailabilityNotifier(props: Props) {
 
   const intl = useIntl()
 
-  const seller = getDefaultSeller(productContext.selectedItem?.sellers) as any
+  const seller = getDefaultSeller(productContext.selectedItem?.sellers) as
+    | any
+    | null
```

Comment:
Isn't this the same thing as doing `as any`? If so, I think you should better type this property to avoid having similar errors in the future.
```suggestion
    Seller
```

Reply:
It would be, but the Seller is not being imported in `1.x` as we are using the `sellerObj` interface